### PR TITLE
doc: update documentation for configuring ACRN RDT features

### PR DESCRIPTION
**Pull request by vijaydhanraj**
Given the recent changes in the way ACRN configures RDT features,
this patch updates the documentation as well to provide clear
guidelines to the user.

Tracked-On: #5063
Signed-off-by: Vijay Dhanraj <vijay.dhanraj@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by dbkinder** [comment]
@NanlinXie Does this look OK to you so I can merge and publish?

File: doc/developer-guides/hld/hv-rdt.rst [hunk 1]
```diff
@@ -38,58 +38,36 @@ IA32_PQR_ASSOC MSR to CLOS 0. (Note that CLOS, or Class of Service, is a
...
-   resources and sets the MAX_PLATFORM_CLOS_NUM. For example, if max CLOS
-   supported by L3 is 16 and L2 is 8, ACRN programs MAX_PLATFORM_CLOS_NUM to
-   8. ACRN recommends consistent capabilities across all RDT
-   resources by using the common subset CLOS. This is done in order to
-   minimize misconfiguration errors.
+   resources as maximum supported CLOS ID. For example, if max CLOS
+   supported by L3 is 16 and MBA is 8, ACRN programs MAX_PLATFORM_CLOS_NUM
+   to 8. ACRN recommends to have consistent capabilities across all RDT
+   resources by using a common subset CLOS. This is done in order to minimize
+   misconfiguration errors.
```
Comment (szhen11) [inline doc/developer-guides/hld/hv-rdt.rst:71]: this logic is for CPD disabled, right? what about CDP enabled?

Comment (vijaydhanraj) [reply doc/developer-guides/hld/hv-rdt.rst:71]: The example mentioned is with CDP disabled but from the code MAX_PLATFORM_CLOS_NUM takes into account CDP vs no CDP before finding the min of all supported RDT CLOS. Please [refer](https://github.com/projectacrn/acrn-hypervisor/blob/8191e1143a1fbba873b0c4a218ceeba5aac2a265/misc/acrn-config/board_config/board_info_h.py#L84) for more details.

Idea was not to confuse users with too much information.

Comment (szhen11) [reply doc/developer-guides/hld/hv-rdt.rst:71]: Thanks to clarify.

**Pull request merged**